### PR TITLE
Camel 14325/add new functions

### PR DESCRIPTION
**Pull request by cherepnalkovski**

### CODE REVIEW

**Comment by cherepnalkovski** [comment]
@oscerd It's already ran with sourcecheck

**Comment by oscerd** [comment]
Nice

File: components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISProducer.java [hunk 1]
```diff
@@ -160,9 +261,11 @@ public void deleteDocument(Exchange exchange) throws Exception {
         Message message = exchange.getIn();
 
         String objectId = message.getHeader(CamelCMISConstants.CMIS_OBJECT_ID, String.class);
+        Boolean allVersions = Boolean.valueOf(message.getHeader(CamelCMISConstants.ALL_VERSIONS, String.class));
```
Comment (davsclaus) [inline components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISProducer.java]: You can just say

message.getHeader(xxx, Boolean.class)

File: components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CamelCMISConstants.java [hunk 2]
```diff
@@ -29,5 +29,7 @@
     String CAMEL_CMIS_RETRIEVE_CONTENT = "CamelCMISRetrieveContent";
     String CAMEL_CMIS_READ_SIZE = "CamelCMISReadSize";
     String CAMEL_CMIS_CONTENT_STREAM = "CamelCMISContent";
+    String ALL_VERSIONS = "allVersions";
```
Comment (davsclaus) [inline components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CamelCMISConstants.java]: CamelCMISAllVersions

File: components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISProducer.java [hunk 3]
```diff
@@ -16,8 +16,11 @@
  */
 package org.apache.camel.component.cmis;
 
+import java.awt.Stroke;
```
Comment (davsclaus) [inline components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CMISProducer.java]: What is this used for smells like it should be removed

**Pull request merged**